app/Http/Requests/TeamRequest.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace App\Http\Requests;
```

Comment:
I really this should be in a `Team` folder and have the name `UpdateRequest`, otherwise it's not clear where it is used.